content/en/docs/getting-started-guides/ubuntu/scaling.md

```diff
@@ -19,27 +19,26 @@ watch -c juju status --color
 {{% /capture %}}
 
 {{% capture steps %}}
-## Kubernetes masters
+## Kubernetes control plane
 
-The provided Kubernetes master nodes act as a control plane for the cluster.
-The deployment has been designed so that these nodes can be scaled independently
-of worker nodes to allow for more operational flexibility.
-To scale a master node up, simply execute:
+The provided deployment has been designed so that the control plane can be scaled independently
+of nodes to allow for more operational flexibility.
+To scale the control plane up, simply execute:
 
     juju add-unit kubernetes-master
 
-This will add another master node to the control plane.
+This will add another control-plane machine to the control plane.
 See the [building high-availability clusters](/docs/admin/high-availability)
 section of the documentation for more information.
 
-## Kubernetes workers
+## Kubernetes nodes
 
-The kubernetes-worker nodes are the load-bearing units of a Kubernetes cluster.
+The nodes are the load-bearing units of a Kubernetes cluster.
 
-By default pods are automatically spread throughout the kubernetes-worker units
+By default pods are automatically spread throughout the node
```

Comment:
`nodes` plural.
also `worker nodes` made sense here, to describe that we are talking about workloads.